test/spec/ui/responsive.spec.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'chai',
+            'ui/responsive'
+        ],
+        function (chai, responsive) {
+            var expect = chai.expect;
+
+            describe('The `responsive` module', function () {
```

Comment:
This test is very weak, but there isn't much else that can be done without mocking out all of jQuery and possibly parts of the DOM itself, which isn't really feasible.